proto/chunk.proto

```diff
@@ -62,6 +62,8 @@ message ChunkRequest {
     optional string location = 11;      // for CreateCloneChunk
     optional string cloneFileSource = 12;   // for write/read
     optional uint64 cloneFileOffset = 13;   // for write/read
+    optional uint64 sendScanMapTimeoutMs = 14;   // for scan chunk
+    optional uint32  sendScanMapRetryTimes= 15;   // for scan chunk
```

Comment:
> does those two items corresponding to
> 
> ```shell
> copyset.scan_timeout_ms=1000
> copyset.scan_retry_times=3
> ```
> 
> in configurations? If so, why not directly read those items from configurations?

Leader read from conf and set into chunkOpRequest pushed to raft, and then follower send scanmap rpc to leader in onApplyFromLog will use it.

Reply:
> > does those two items corresponding to
> > ```shell
> > copyset.scan_timeout_ms=1000
> > copyset.scan_retry_times=3
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > in configurations? If so, why not directly read those items from configurations?
> 
> Leader read from conf and set into chunkOpRequest pushed to raft, and then follower send scanmap rpc to leader in onApplyFromLog will use it.

Yep, I think the follower can just use those configurations from the config file rather than from rpc